src/openrct2/rct2/S6Importer.cpp

```diff
@@ -187,6 +187,18 @@ class S6Importer final : public IParkImporter
         return false;
     }
 
+    static bool IsLikelyUtf8(const std::string_view s)
+    {
+        for (auto c : s)
+        {
+            if (static_cast<uint8_t>(c) >= 128)
```

Comment:
This is wrong. CP1252 has plenty of characters >= 128. It will thus detect numerous CP1252 strings and (virtually) all non-UTF8 multibyte ones as UTF-8.

The old code checked for codepoints between FORMAT_COLOUR_CODE_START and FORMAT_COLOUR_CODE_END, because these don't typically appear in scenario titles and descriptions.

Reply:
Done, and improved further by also checking for arg format codes.